lib/internal/Magento/Framework/App/Response/HttpInterface.php

```diff
@@ -16,4 +16,49 @@
      * @return void
```

Comment:
`\Magento\Framework\HTTP\PhpEnvironment\Response` have public methods which were not added to current interface. Does it make sense to add methods like `clearHeader` or `getHttpResponseCode` to it?